com.unity.ml-agents/Runtime/SideChannel/EngineConfigurationChannel.cs

```diff
@@ -1,13 +1,14 @@
 using System.IO;
+using System;
 using UnityEngine;
 
 namespace MLAgents
 {
     public class EngineConfigurationChannel : SideChannel
     {
-        public override int ChannelType()
+        public EngineConfigurationChannel()
         {
-            return (int)SideChannelType.EngineSettings;
+            ChannelId = new Guid("e951342c-4f7e-11ea-b238-784f4387d1f7");
```

Comment:
Should the strings or the Guids be static/const members of the class instead? Not sure how to make it feel less magic.

Reply:
Just curious, where did these come from? Totally random, or did you use e.g. uuid3 or uuid5 (https://docs.python.org/3/library/uuid.html#uuid.uuid3)?